### ISSUE TITLE
from poll: overview of popular npm modules

### ISSUE THREAD

**Issue by sidorares** [sidorares/melbnodejs#3]
"I will like to hear about popular modules people are using for production apps. Beyond the usual Async, request and Express."
- I'd probably make it 20% time overview what is on npm and 80% time "what search tools are available? What metrics are good to decide which package to use when there are 10 of them with similar functionality?"

http://stackoverflow.com/questions/10568512/how-to-find-search-find-npm-packages

Maybe quick workshop on "how to make chunk of my JS code worth publishing and upload to npm" could fit this talk (or complement it as a second talk)

**Comment by rkstedman** [comment]
Talk could also compare publishing private vs public npm modules

**Comment by sidorares** [comment]
@rkstedman "publishing private" == "using git repo (branch,tag) as npm dependency"?

**Comment by rkstedman** [comment]
@sidorares That is definitely one way to do it, but I was thinking of the alternative method of deploying a private npm registry to enable sharing and use of the private modules within a team, without registering them to the global npmjs.org. Using this method, as opposed to a git repo, also allows a company to provide governance around use of public npm module versions to ensure stability and perform audits of modules in use. StrongLoop blogged about using node-reggis for this purpose: http://blog.strongloop.com/deploy-a-private-npm-registry-without-couchdb-or-redis/

**Comment by sidorares** [comment]
Yes, I'm interested in "private npm registry" talk. Don't have any experience myself. We had local npm registry at campjs, I think @eugenware knows well how to install/configure one

**Comment by ghost** [comment]
would be a very interesting talk :+1:
